Automattic/wp-calypso
# DevDocs: Use `LazyRender` instead of `DelayRender`

Loading all of the blocks and components in the DevDocs can take a while, so a `timeout` modification of `LazyRender` (`DelayRender`) has been used to load the first 20 components immediately, and then all remaining components after a timeout. 

With the search component available to query the entire set of examples, we can instead lazy load smaller chunks of components as needed, without worrying about the timeout and scaling as more components are added.

**To test:**
- Visit http://calypso.localhost:3000/devdocs/design
- The initial placeholder time is a result of the unrelated `AsyncLoad`
- After the initial 10 components load:
- Search for an un-rendered component (i.e. "Wizard" or "Version") to verify that they're immediately available in the search results.
- Reload /devdocs/design and scroll through the page - components should be loaded with little delay.
